### PR TITLE
made changes based on suggestions

### PR DESCRIPTION
**Pull request by shry678**

### CODE REVIEW

**Review by kellymckinnon — APPROVED** [review]
great, thanks!

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -64,10 +65,20 @@
...
+            android:authorities="com.facebook.app.FacebookContentProvider443391136243627"
             android:exported="true" />
-    </application>
 
+        <service android:name=".MyFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+            </intent-filter>
+        </service>
+        <!--<service android:name=".FirebaseIDService">-->
```
Comment (kellymckinnon) [inline app/src/main/AndroidManifest.xml]: did you mean to leave this commented out?

Comment (shry678) [reply app/src/main/AndroidManifest.xml]: yes, still working on that part!! :)

Comment (kellymckinnon) [reply app/src/main/AndroidManifest.xml]: ok, cool :)

**Pull request merged**